### ISSUE TITLE
"Signal" feature doesn't compile

### ISSUE THREAD

**Issue by GildedHonour** [actix/actix-web#37]
#....................
    actix = "^0.3.5"
    actix-web = { git = "https://github.com/actix/actix-web.git", version = "0.3.0",  features = ["signal"] }

    //....................
    use actix_web::*;

    #[cfg(unix)]
    use actix::actors::signal::{ProcessSignals, Subscribe};

    //....................
    let signals = Arbiter::system_registry().get::<ProcessSignals>();
    signals.send(Subscribe(addr.subscriber()));

===============>

    error[E0433]: failed to resolve. Use of undeclared type or module `Arbiter`
      --> src/main.rs:73:19
       |
    73 |     let signals = Arbiter::system_registry().get::<ProcessSignals>();
       |                   ^^^^^^^ Use of undeclared type or module `Arbiter`

**Comment by fafhrd91** [comment]
it is not possible to say why it does not compile from this issue description. I need more info

**Comment by fafhrd91** [comment]
here is hello world example, it doesn't use signals 

https://github.com/actix/actix-web/tree/master/examples/hello-world

**Comment by GildedHonour** [comment]
from https://actix.github.io/actix-web/guide/qs_3_5.html


```

fn main() {
    let sys = actix::System::new("signals");

    let addr = HttpServer::new(|| {
        Application::new()
            .resource("/", |r| r.h(httpcodes::HTTPOk))})
        .bind("127.0.0.1:8080").unwrap()
        .start();

    // Subscribe to unix signals
    let signals = Arbiter::system_registry().get::<ProcessSignals>();
    signals.send(Subscribe(addr.subscriber()));

    println!("Started http server: 127.0.0.1:8080");
    let _ = sys.run();
}
```


====>

```
error[E0433]: failed to resolve. Use of undeclared type or module `Arbiter`
  --> src/main.rs:94:19
   |
94 |     let signals = Arbiter::system_registry().get::<ProcessSignals>();
   |                   ^^^^^^^ Use of undeclared type or module `Arbiter`

error: aborting due to previous error

```

**Comment by GildedHonour** [comment]
I need signals because each time I shut down my server I then have "address is already in use" for the next 2 minutes

**Comment by GildedHonour** [comment]
how to turn off this 2 minutes feature, at least in the  development ?

**Comment by fafhrd91** [comment]
I'll fix "address in use" later today.
for now use `.keep_alive(Some(0))`

```rust
let addr = HttpServer::new(|| {
        Application::new()
            .resource("/", |r| r.h(httpcodes::HTTPOk))})
        .bind("127.0.0.1:8080").unwrap()
        .keep_alive(Some(0))                     // <- add this
        .start();
```

**Comment by GildedHonour** [comment]
`keep_alive(Some(0))` isn't working, the address is still in use after I've shut down my server

**Comment by fafhrd91** [comment]
ok, let me check

**Comment by fafhrd91** [comment]
should be fixed in 1f7aee2

I will add better fix for "address in use" a bit later. it will work without signals
